atom/browser/browser_linux.cc

```diff
@@ -89,7 +89,7 @@ bool Browser::RemoveAsDefaultProtocolClient(const std::string& protocol,
 bool Browser::SetAsDefaultProtocolClient(const std::string& protocol,
                                          mate::Arguments* args) {
   std::string desktop_name;
-  if (!args->GetNext(&desktop_name)) {
+  if (args && !args->GetNext(&desktop_name)) {
```

Comment:
args should always exist, where are you seeing it as null?

Reply:
`WebContents::OnRegisterProtocol` in `atom/browser/api/atom_api_web_contents.cc`

Reply:
https://github.com/brave/muon/pull/309 was merged into 4.4 so this isn't a cr62 issue